charts/firefly/templates/_helpers.tpl

```diff
@@ -48,6 +48,13 @@ Create chart name and version as used by the chart label.
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
 {{- end }}
 
+{{/*
+Create Firefly Node Name
+*/}}
+{{- define "firefly.nodeName" -}}
+{{- printf "%s" .Release.Namespace | replace "+" "_" | trunc 63 | trimSuffix "-" }}
```

Comment:
Maybe should be `%s-%s` with `.Release.Namespace` and `(include .firely.fullname .)`? 

Thinking of when one might install multiple FFs releases in the same namespace for test use cases.